tests/functional/aws-node-sdk/test/bucket/getLocationTemp.js

```diff
@@ -0,0 +1,55 @@
+import assert from 'assert';
```

Comment:
the commit that introduced this test should not be forward ported.  this test was just for 6.4.